pkgs/servers/sql/cockroachdb/deps.nix

```diff
@@ -95,17 +95,17 @@
     fetch = {
       type = "git";
       url = "https://github.com/cockroachdb/cockroach";
```

Comment:
@kamilchm I used version `1.1.0`. I hadn't noticed there is new release. Thanks!